lib/zlib.js

```diff
@@ -35,7 +35,8 @@ const {
     isAnyArrayBuffer,
     isArrayBufferView
   }
-} = require('util');
+} 
+= require('internal/util');
```

Comment:
And one more nit: there shouldn't have newline.

![Screen Shot 2019-03-19 at 8 47 27 AM](https://user-images.githubusercontent.com/23313266/54572639-9aa00580-4a23-11e9-871f-ca33a9b07d47.png)